backend/naming.py

```diff
@@ -90,12 +90,13 @@ def _get_formatting_data(volume_id: int, issue_id: int=None) -> dict:
 		clean_title = volume_data.get('title') or 'Unknown'
 	
 	formatting_data = {
-		'series_name': (volume_data.get('title') or 'Unknown').replace('/', '').replace(r'\\', ''),
+		'series_name': volume_data.get('title', 'Unknown').replace('/', '').replace(r'\\', ''),
 		'clean_series_name': clean_title.replace('/', '').replace(r'\\', ''),
-		'volume_number': volume_data.get('volume_number') or 'Unknown',
-		'comicvine_id': volume_data.get('comicvine_id') or 'Unknown',
-		'year': volume_data.get('year') or 'Unknown',
-		'publisher': volume_data.get('publisher') or 'Unknown'
+		'volume_number': volume_data.get('volume_number', 'Unknown'),
+		'comicvine_id': volume_data.get('comicvine_id', 'Unknown'),
+		'year': volume_data.get('year', 'Unknown'),
+		'publisher': volume_data.get('publisher', 'Unknown'),
+		'issues_as_volumes': bool(volume_data.get('issues_as_volumes', False))
 	}
```

Comment:
The reason `or` is used instead of the default value in the `get` function is that this way non-`None` values that are still invalid are also caught. For example, if the series_name is `""`, normally this would be replaced with `Unknown`, but with your implementation it will be left as-is because it isn't `None`, so allowed. Same with the year: sometimes people set the year to `0` if they don't know it, and the `or` would replace this with `Unknown` while your implementation would leave it to `0`.

So please remove these changes.

Reply:
no problem. i'll keep the issues_as_volumes as above ok?

Reply:
Yes that's okay